### PR TITLE
Add vertical and horizontal offset values.

### PR DESCRIPTION
**Pull request by cdata**
/cc @morethanreal

### CODE REVIEW

File: iron-dropdown.html [hunk 1]
```diff
@@ -99,6 +99,26 @@
           },
 
           /**
+           * A pixel value that will be added to the position calculated for the
```
Comment (notwaldorf) [inline iron-dropdown.html]: super nitty, but I would rephrase this as "A pixel value that will be added to the horizontal position of the dropdown, in the direction given by `horizontalAlign`". Or something like that (too much?)

File: iron-dropdown.html [hunk 2]
```diff
@@ -190,6 +210,8 @@
             target = this._positionRect.left;
           }
 
+          target += this.horizontalOffset;
```
Comment (notwaldorf) [inline iron-dropdown.html]: Hmm so it should be negative for `left`? Maybe that should go in the comment too :(

File: iron-dropdown.html [hunk 3]
```diff
@@ -202,6 +224,8 @@
             target = this._positionRect.top;
           }
 
+          target += this.verticalOffset;
```
Comment (notwaldorf) [inline iron-dropdown.html]: samesies

**Comment by notwaldorf** [comment]
LGTM, but I've got minor objections to the comments 🎢

File: iron-dropdown.html (hunk 1 above)
Comment (cdata) [inline iron-dropdown.html]: This isn't quite correct, since the value will be added in the same direction regardless of the value of `horizontalAlign`. For example, if I `horizontalAlign` to the `"left"`, with a `horizontalOffset` of `10`, I will add `10` to the final horizontal position. If I `horizontalAlign` to the `"right"` instead, it will still add `10` to the final horizontal position. You would have to change it to `-10` to actually change the direction.

File: iron-dropdown.html (hunk 2 above)
Comment (cdata) [inline iron-dropdown.html]: Yeah. Sorry, looks like you figured out what I explained above.

File: iron-dropdown.html (hunk 3 above)
Comment (cdata) [inline iron-dropdown.html]: Word.
